src/client/app/admin/authentication/inactive.component.ts

```diff
@@ -1,16 +1,17 @@
 import { Component } from '@angular/core';
-import { User } from '../user.class';
+
+import { User } from '../user';
+
 import { AuthenticationService } from './authentication.service';
 
 @Component({
-	templateUrl: './inactive.component.html'
+	templateUrl: './inactive.component.html',
+	providers: [ User ]
 })
-
 export class InactiveComponent {
-	user: User;
-
 	constructor(
-		private auth: AuthenticationService
+		private auth: AuthenticationService,
+		private user: User
```

Comment:
This was originally when I was trying to figure out why User/AuthenticationService was undefined during runtime.